mp/game/momentum/resource/momentum_english_ref.res

```diff
@@ -367,6 +369,18 @@
         "MOM_Settings_Online_Trails" "Enable online ghost trails"
         "MOM_Settings_Online_Trails_TT" "When enabled, other players can have a trail showing their paths"
         
+        // Rocket Jump settings
+        "MOM_Settings_Tab_RJ" "RJ"
+        "MOM_Settings_RJ_None" "None"
+        "MOM_Settings_RJ_Momentum" "Momentum"
```

Comment:
This token already exists as `"GameUI2_LogoLeft"`, pull it out into a shared `MOM_Momentum` token, ensure that wherever uses `GameUI2_LogoLeft` is also updated to use this new token.

Reply:
I only found one reference to it in MainMenu.cpp. Is this the only one or am I missing any others?

Reply:
I think that's the only one yeah.